alternator/stats.hh

```diff
@@ -102,10 +101,15 @@ public:
     uint64_t wcu_total[NUM_TYPES] = {0};
     // CQL-derived stats
     cql3::cql_stats cql_stats;
-private:
-    // The metric_groups object holds this stat object's metrics registered
-    // as long as the stats object is alive.
+};
+
+void register_metrics(seastar::metrics::metric_groups& metrics, stats& stats);
+void register_metrics(seastar::metrics::metric_groups& metrics, stats& stats, const sstring& ks, const sstring& table);
```

Comment:
I know that function overloading is a thing in C++, but I find it confusing. Could we name the functions differently? Perhaps register_global_metrics and register_per_table_metrics or something?

Reply:
sure, I'll do that